src/components/TaskCard/TaskCard.tsx

```diff
@@ -35,7 +36,7 @@ function FiledRequestCard({ filedRequestCardData, onDelete }: FiledRequestCard)
             <Box sx={styles.card__body}>
                 <Box sx={{ width: '60%' }}>
                     <Box>{description}</Box>
-                    <Box>{deadline}</Box>
+                    <Box>{deadline != null ? deadline.toString() : ''}</Box>
```

Comment:
null evaluates to false in js, this line can be `deadline && deadline.toString()` instead of the ternary, you can also put the conditional around the box which would be better `{deadline && <Box>...</Box>}`